lib/chai/utils/getOperator.js

```diff
@@ -0,0 +1,67 @@
+/*!
+ * Module dependencies
+ */
+
+var inspect = require('./inspect');
+var config = require('../config');
+var flag = require('./flag');
+
+function isObjectType(obj) {
+  var str = inspect(obj),
+    type = Object.prototype.toString.call(obj);
```

Comment:
IMO we should use [`type-detect`](https://github.com/chaijs/type-detect) here for the sake of consistency.